.bazelrc

```diff
@@ -414,3 +414,5 @@ build:windows --dynamic_mode=off
 try-import %workspace%/clang.bazelrc
 try-import %workspace%/user.bazelrc
 try-import %workspace%/local_tsan.bazelrc
+
+info --ui_event_filters=-WARNING
```

Comment:
i dont think we want to filter all warnings (for info)

im wondering if instead of suppressing the warnings we could prevent them

Reply:
I think we do not miss anything import:
```
root:[envoy]$ bazel info &>old
root:[envoy]$ bazel info --ui_event_filters=-WARNING &>new
root:[envoy]$ diff old new
1d0
< WARNING: info command does not support starlark options. Ignoring options: [--@com_googlesource_googleurl//build_config:system_icu=0]
9,10c8,9
< gc-count: 15
< gc-time: 272ms
---
> gc-count: 16
> gc-time: 328ms
23c22
< used-heap-size: 35MB
---
> used-heap-size: 36MB
```

Reply:
when i run the command with this flag i lose deprecation warnings - i dont think warnings should be supressed - how about trying to resolve the problem itself

/wait